src/layout/SigningStatusPanel/SigningStatusPanel.module.css

```diff
@@ -0,0 +1,42 @@
+.loadingContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.buttonLink {
+  color: white;
+  text-decoration: none;
+}
+
+.signingPanel {
+  padding-top: var(--fds-spacing-8);
+  display: flex;
+  flex-direction: column;
+
+  .contentContainer {
+    padding: var(--fds-spacing-2); /* TODO: remove when MUI Grid is removed?*/
```

Comment:
MUI grid has been removed! :partying_face:

Reply:
I realise that it still has the same functionality with padding and stuff, so still need this to make it look right :/